css/src/help-center-metabox.scss

```diff
@@ -1,20 +1,18 @@
 .postbox {
 	.yoast-help-center {
-		.yoast-help-center {
-			&__button {
-				border:          none;
-				box-shadow:      none;
-				text-transform:  none;
-				text-decoration: underline;
-				color:           #0073aa;
-				margin:          0;
-				min-height:      0;
-				font:            inherit;
-				padding:         8px 0;
+		&__button {
```

Comment:
Sorry what I meant was:
```
...
.yoast-help-center {
    .yoast-help-center_button {
...
```

As the button should be only used when present inside the `.yoast-help-center` div to begin with.